src/Web/Hyperbole/HyperView/Forms.hs

```diff
@@ -235,6 +236,16 @@ checkbox isChecked = do
   tag "input" @ att "type" "checkbox" . name nm $ none @ checked isChecked
 
 
+
+radio :: ToParam a => a -> Bool -> View (Input id a) ()
+radio val isChecked = do
+  Input (FieldName nm) <- context
+  let (ParamValue valTxt) = toParam val
```

Comment:
I've used `ToParam` here and not `ToJSON` because `gFormParse` uses `FromParam`.

Reply:
Good, forms are url encoded, so that makes sense